features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/index.jsx

```diff
@@ -262,11 +262,20 @@ class Details extends React.Component {
      * @memberof Details
      */
     componentDidMount() {
+        this.updateActiveLink();
+        this.updateSubscriptionData();
+    }
+
+    /**
+     *
+     * Selects the active link for the side panel based on the URL
+     * @memberof Details
+     */
+    updateActiveLink() {
         const currentLink = this.props.location.pathname.match(/[^\/]+(?=\/$|$)/g);
-        if (currentLink && currentLink.length > 0) {
+        if (currentLink && currentLink.length > 0 && this.state.active !== currentLink[0]) {
```

Comment:
`this.state.active` should have unloaded to a constant beforehand

```javascript
const {active} = this.state;
```

Reply:
Fixed